src/api/CortexClient.ts

```diff
@@ -279,4 +282,23 @@ export class CortexClient implements CortexApi {
 
     return responseBody;
   }
+
+  private async fetchAuthenticated(input: RequestInfo, init?: RequestInit): Promise<Response> {
+    let token: string | undefined = undefined;
```

Comment:
i think you could get rid of the let here if you do: 

    const { token }: string | undefined = this.identityApi !== undefined ? await this.identityApi.getCredentials() : undefined;